src/Shared/ParameterBindingMethodCache.cs

```diff
@@ -272,6 +279,27 @@ static bool ValidateReturnType(MethodInfo methodInfo)
         }
     }
 
+    private static MethodInfo? GetIBindableFromHttpContextMethod(Type type)
+    {
+        // Check if parameter is bindable via static abstract method on IBindableFromHttpContext<TSelf>
```

Comment:
Thoughts on this approach of checking for and calling `static abstract` generic methods @davidfowl @halter73?

Reply:
Looks good to me. I suspect @davidfowl won't like the linq and will recommend just returning out of a foreach.

Reply:
Yah I literally started with that and then the appeal of terseness got to me.